index.html

```diff
@@ -180,6 +180,45 @@ <h2 class="style">Multi-Post Stories</h2>
       </div>
     </section>
   </section>
+  <section class="contact-form">
+    <header class="contact-us">
+      <h1 class="style">Contact US</h1>
+    </header>
+    <p class="style">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+    <form class="form-outlay" method="POST" action="https://formspree.io/f/mvodvrgn">
+      <fieldset class="contact-form-fieldset">
+        <ul class="contact-form-ul">
+          <li><input class="input-text style" type="text" name="user_name" placeholder="Text input" required maxlength="30"></li>
+          <li><input class="input-email style" type="email" name="user_email" placeholder="jeremy@gmail.com" required ></li>
+          <li><textarea class="style" required maxlength="500">Enter text here</textarea></li>
+          <li><button class="see-project submit-button" type="submit">See Project</button></li>
+        </ul>        
```

Comment:
### OPTIONAL
- [ ] You don't have to wrapped by list when you create a form. [form-elements](https://www.w3schools.com/html/html_forms.asp)

Reply:
Thank you for the comments and feedback. I surely will improve my commit messages in subsequent projects 👍